README.md

```diff
@@ -27,7 +27,7 @@ By the end of this lesson, students should be able to:
 The DOM is a (potentially) large object that describes the structure of our
 content. Since it's an object, we can use normal techniques to get and set data!
 In the browser, the DOM is represented by the `document` object. JS specifies
-some built-in methods that make using the DOM easier. Remember! The DOM
+some built-in methods that make using the DcoOM easier. Remember! The DOM
```

Comment:
I believe this is a typo. Did you inspect changes with `git diff` before committing? :smile: